autoload/codi.vim

```diff
@@ -359,8 +359,8 @@ endfunction
 
 " Trigger autocommands and silently update
 function! codi#update()
-  " Bail if no codi buf to act on
-  if !s:get_codi('bufnr') | return | endif
+  " Bail if no codi buf to act on except running on nvim
```

Comment:
This comment is incorrect. It should be if virtual text is enabled and available.